doc/source/whatsnew/v1.2.0.rst

```diff
@@ -49,6 +49,42 @@ For example:
    buffer = io.BytesIO()
    data.to_csv(buffer, mode="w+b", encoding="utf-8", compression="gzip")
 
+Arithmetic with Timestamp and Timedelta-based Intervals
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Arithmetic can now be performed on :class:`Interval` s having their left and right
+ends as :class:`Timestamp`s or :class:`Timedelta` s, like what would be possible
+if the ends were numeric (:issue:`35908`).
+Before this change, it could be performed in an indirect way like this:
+
+.. ipython:: python
```

Comment:
I feel like we don't need this here, the new method is better

Reply:
Sure, makes sense.